dev/packages/alpha/nginx/0.0.3/dataset/access/fields/fields.yml

```diff
@@ -0,0 +1,11 @@
+- name: nginx.access
+  type: group
+  fields:
+  - name: remote_ip_list
+    type: array
+    description: |
+      An array of remote IP addresses. It is a list because it is common to include, besides the client IP address, IP addresses from headers like `X-Forwarded-For`. Real source IP is restored to `source.ip`.
+  - name: user_agent
```

Comment:
I wonder if this group of fields is still used or obsolete because it is now all ECS? Same for the geoip.

Reply:
There are some magic steps around this in the ECS, so I assume it's used?
e.g.:
```
- geoip:
    database_file: GeoLite2-ASN.mmdb
    field: source.ip
    target_field: source.as
    properties:
    - asn
    - organization_name
    ignore_missing: true
```